src/fmu/ensemble/ensembleset.py

```diff
@@ -744,6 +775,10 @@ def get_wellnames(self, well_match=None):
             summary file or no matched well names.
 
         """
+        warnings.warn(
+            "fmu.ensemble.etc is deprecated and will be removed in later versions.",
+            FutureWarning,
+        )
```

Comment:
Not sure if it is supposed to be deprecated or not, but the message is at least wrong.
```suggestion
        warnings.warn(
            "ensembles.get_wellnames() is deprecated and will be removed in later versions.",
            FutureWarning,
        )
```

Reply:
fixed